libraries/lib-ffmpeg-support/wrappers/AVIOContextWrapper.cpp

```diff
@@ -56,6 +57,8 @@ AVIOContextWrapper::~AVIOContextWrapper()
 
    if (mFFmpeg->avio_context_free != nullptr)
       mFFmpeg->avio_context_free(&mAVIOContext);
+   else
+      mFFmpeg->av_free(mAVIOContext);
```

Comment:
This was causing me issues on Windows, but looks like that was a different bug! Thank you for noticing!